dashboard/src/components/InstanceListView/index.test.tsx

```diff
@@ -0,0 +1,29 @@
+import { shallow } from "enzyme";
+import * as React from "react";
+
+import { MessageAlert } from "../ErrorAlert";
+import { InstanceListView } from "./index";
+
+it("renders the warning for alpha feature", () => {
+  const wrapper = shallow(
+    <InstanceListView
```

Comment:
you may be able to simplify this by casting a smaller object to the InstanceListViewProps (e.g. https://github.com/kubeapps/kubeapps/blob/ea625419021199bc82a1365d75d5ab60c0660fa7/dashboard/src/components/SchemaForm/FieldTemplate.test.tsx#L8), we may need to export InstanceListViewProps.

Reply:
I could omit some of the parameters but not all of them. Thanks!